spec/acceptance/tests/command_supported_throw.rb

```diff
@@ -0,0 +1,23 @@
+test_name 'PowerShell Module - should catch and rethrow exceptions up to puppet'
+confine :to, :platform => 'windows'
+
+pexception = <<-MANIFEST
+  exec{'PowershellException':
+    provider  => powershell,
+    command   => 'throw "We are writing an error"',
+  }
+MANIFEST
+
+agents.each do |agent|
+  opts = {
+    # TODO: with this on like before, the test doesn't work right
+    # :catch_failures => true,
+    :future_parser => (ENV['FUTURE_PARSER'] != 'false'),
+    # TODO: is this the right exit code?? maybe remove this given catch_failures => true
+    :acceptable_exit_codes => [1]
```

Comment:
Yeah, was looking through that already... wanted your opinion on idiomatic test writing for QA.  Will change per your recommendation.

Thanks